gulpfile.js

```diff
@@ -335,12 +335,23 @@ gulp.task('deploy-s3', function(done) {
 
 });
 
+function getCredentials() {
```

Comment:
I had tried `promisify` without the context object and got errors even when I had the credentials, so I think the context is necessary.